### PR TITLE
Feature/word counting

### LINKED ISSUE #8: Feature 1: Count number of words in a text file
Display the number of words in a text file.

### LINKED ISSUE #12: Feature 1: word count test coverage
Add unit tests under tests package

### PR DESCRIPTION
**Pull request by QinAlex**
After I read your comments I all fix the parts you point it out. Thanks for pointing out!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# COSC499-Team-Git
+# Feature/ Word-Counting
```
Comment (rachellegelden) [inline README.md]: This is not needed here. The README.md file is not used to describe a specific to a branch. It is used to describe the repository, so we want to keep it general

File: src/tests/CountWordsTest.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+				
+		// Given text
+		String givenText = "haha lol wut is that ";
+				
+		// When the input string is given
+		int wordCount = CountWords.countWords(givenText);
+
+		// Then
+		assertEquals(expectedResult, wordCount);
+	}
```
Comment (rachellegelden) [inline src/tests/CountWordsTest.java:56]: Looks great !!! 🥇 💯

**Review by rachellegelden — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (QinAlex) [reply README.md]: Sorry for the miss-understood for the ReadMe file.

**Review by rachellegelden — APPROVED** [review]
Approved 🥇

**Pull request merged**